src/main/java/org/cactoos/map/MapEnvelope.java

```diff
@@ -131,4 +132,8 @@ public final void clear() {
         return this.map.value().entrySet();
     }
 
+    @Override
```

Comment:
@SharplEr what about ``MapOf``? Does it inherit ``toString()`` from this ``MapEnvelope``?

Reply:
Yes, it is. I use this `toString()` in the test for MapOf for example.